### PR TITLE
update OMBInfo

### PR DESCRIPTION
**Pull request by rianfowler**
## Testing done

**shows new content**
burials: http://localhost:3001/burials-and-memorials/pre-need/form-10007-apply-for-eligibility/introduction
education: http://localhost:3001/education/submit-school-feedback/introduction
healthcare: http://localhost:3001/health-care/apply/application/introduction

**shows old content**
http://localhost:3001/education/apply-for-education-benefits/application/5490/introduction

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/hca/containers/IntroductionPageGated.jsx [hunk 1]
```diff
@@ -114,7 +116,13 @@ const LoggedOutContent = connect(
       downtime={route.formConfig.downtime}
     />
     <div className="omb-info--container" style={{ paddingLeft: '0px' }}>
-      <OMBInfo resBurden={30} ombNumber="2900-0091" expDate="05/31/2018" />
+      {environment.isProduction() ? (
+        <OMBInfo resBurden={30} ombNumber="2900-0091" expDate="05/31/2018" />
+      ) : (
+        <OMBInfo resBurden={30} ombNumber="2900-0091" expDate="05/31/2018">
+          <HealthcareModalContent resBurden={30} ombNumber="2900-0091" />
```
Comment (rianfowler) [inline src/applications/hca/containers/IntroductionPageGated.jsx]: link

Comment (erikphansen) [reply src/applications/hca/containers/IntroductionPageGated.jsx]: "link"?

File: src/applications/hca/containers/IntroductionPageGated.jsx [hunk 2]
```diff
@@ -114,7 +116,13 @@ const LoggedOutContent = connect(
       downtime={route.formConfig.downtime}
     />
     <div className="omb-info--container" style={{ paddingLeft: '0px' }}>
-      <OMBInfo resBurden={30} ombNumber="2900-0091" expDate="05/31/2018" />
+      {environment.isProduction() ? (
```
Comment (erikphansen) [inline src/applications/hca/containers/IntroductionPageGated.jsx]: adding the gating logic here is fine, but this component itself is gated in the form config.

Comment (rianfowler) [reply src/applications/hca/containers/IntroductionPageGated.jsx]: that's fine- I don't want it on production. I'll add it to both components once we go to production if it's still necessary

File: src/applications/hca/containers/IntroductionPageGated.jsx (hunk 1 above)
Comment (rianfowler) [reply src/applications/hca/containers/IntroductionPageGated.jsx]: yeah I needed to link to this in another PR and AFAIK there's not another easy way .

**Comment by rianfowler** [comment]
plan is to roll this out to staging for a final design / content review. i'll do another PR to fix the rest of the forms.

**Review by jbalboni — APPROVED** [review]

**Comment by jbalboni** [comment]
@rianfowler I think you need to merge the latest (if you haven't) and re-run the build to get around the Drupal errors.

**Pull request merged**